config/config_defaults.go

```diff
@@ -805,6 +804,17 @@ const CGRATES_CFG_JSON = `
 },
 
 
+"prometheus_agent": {
+	"enabled": false,			// enables the prometheus agent: <true|false>
+	"path": "/prometheus",			// endpoint for prometheus metrics
+	"collect_go_metrics": false, 		// include Go runtime metrics (memory, GC, goroutines)
+	"collect_process_metrics": false, 	// include process metrics (CPU, file descriptors)
+	// "cores_conns": [], 			// connections to CoreS, empty to disable: <""|*internal|$rpc_conns_id>
```

Comment:
If this is not nil and empty by default, you have to keep it uncommented in defaults.